chia/harvester/harvester_api.py

```diff
@@ -95,6 +96,8 @@ async def new_signage_point_harvester(
         start = time.monotonic()
         assert len(new_challenge.challenge_hash) == 32
 
+        plot_strength = calculate_plot_strength(self.harvester.constants, new_challenge.last_tx_height)
```

Comment:
i would change to calculate_requierd_strength since the plot strength itself is fixed in the plot and returned in the proof bytes, i think this name is confusing